03-MVC-AngularJS/module.js

```diff
@@ -0,0 +1,113 @@
+var myApp = angular.module('myApp', []);
+
+myApp.factory('Storage', function () {
+	return {
+		get: function(string) {
+			var json_obj = localStorage.getItem(string);
+			return JSON.parse(json_obj);
+		},
+		post: function(list, string) {
```

Comment:
I think `set` is better than `post` when you are dealing with storage and not with http services